Peer.GitHub/GraphQL/PullRequestSearch/SearchParams.cs

```diff
@@ -31,6 +34,7 @@ public class SearchParams
                 throw new ArgumentOutOfRangeException(nameof(pageSize), "Must be between 1 and 100");
 
             PageSize = pageSize;
+            EndCursor = endCursor;
```

Comment:
I'd have to look at docs again but I think end-cursor is the field that the cursor value comes to us in, in this case it's the cursor we want to start at. I can't imagine we'd do a query from a cursor other than the end-cursor of the previous page but there might be a more context-appropriate name.

Reply:
I'll noodle on that

Reply:
A few options come to mind (in vaguely the order of my preference):
<dl>
<code>after</code><br>
<code>nextPage</code><br>
<code>page</code><br>
<code>pageCursor</code><br>
<code>nextPageCursor</code><br>
<code>afterCursor</code><br>
<code>nextCursor</code><br>
<code>startCursor</code><br>
</dl>

They all have a bit of a stink to them, though `endCursor` is my least favorite by a small margin since it's named after the source of the value rather than for what the field is.

I like `after` the most because that's the name of the query field it populates. Next to that `nextPage` seems like the best description of what the value means.

I don't really like `startCursor` though I'm not sure why. It is the cursor that specifies where to start so it makes sense; maybe I just don't like that the response we got it from and the response we're expecting will (or at least could) have `startCursor` values and neither one will match the value of the property.

The rest all seem like okay things to call the thing we _know_ we're talking about, but I don't think they'd be as self-explanatory as good names can be.

Reply:
I'll do after

Reply:
done